### PR TITLE
chore: adding multitenancy conflist in azure-cni

### PR DESCRIPTION
**Pull request by pjohnst5**
Adding in the multitenancy conflists into the Azure-CNI image

### CODE REVIEW

**Comment by pjohnst5** [comment]
This is for testing mainly

File: cni/windows.Dockerfile [hunk 1]
```diff
@@ -21,6 +21,7 @@ COPY --from=azure-vnet /azure-container-networking/cni/azure-$OS.conflist /paylo
 COPY --from=azure-vnet /azure-container-networking/cni/azure-$OS-swift.conflist /payload/azure-swift.conflist
 COPY --from=azure-vnet /azure-container-networking/cni/azure-$OS-swift-overlay.conflist /payload/azure-swift-overlay.conflist
 COPY --from=azure-vnet /azure-container-networking/cni/azure-$OS-swift-overlay-dualstack.conflist /payload/azure-swift-overlay-dualstack.conflist
+COPY --from=azure-vnet /azure-container-networking/cni/azure-$OS-multitenancy-transparent-vlan.conflist /payload/azure-multitenancy-transparent-vlan.conflist
```
Comment (rbtr) [inline cni/windows.Dockerfile:24]: does `azure-linux-multitenancy-transparent-vlan.conflist` exist?

Comment (pjohnst5) [reply cni/windows.Dockerfile:24]: Yeah:
https://github.com/Azure/azure-container-networking/blob/a19e5fb5cd824d7ff8888c079b9bf1eac8c4a6aa/cni/azure-linux-multitenancy-transparent-vlan.conflist

Comment (pjohnst5) [reply cni/windows.Dockerfile:24]: What say ye @rbtr?

Comment (rbtr) [reply cni/windows.Dockerfile:24]: don't only add it to the Windows Dockerfile then

Comment (pjohnst5) [reply cni/windows.Dockerfile:24]: It already is
https://github.com/Azure/azure-container-networking/blob/8b5a956db8181de0099a75d71f8dc346fa44ca7a/cni/linux.Dockerfile#L22C5-L22C158

Comment (pjohnst5) [reply cni/windows.Dockerfile:24]: It's in the linux dockerfile already I mean @rbtr

**Review by rbtr — DISMISSED** [review]

**Comment by rbtr** [comment]
@pjohnst5 can we take https://github.com/Azure/azure-container-networking/pull/2559 instead?

**Comment by rbtr** [comment]
> @pjohnst5 can we take #2559 instead?

@pjohnst5 bump^

**Comment by pjohnst5** [comment]
@rbtr yes, thanks, I'll close this one

**Comment by pjohnst5** [comment]
Hi @rbtr, can we merge this in the meantime while #2559 is in progress?

**Comment by pjohnst5** [comment]
Closing in lieu of #2617

**Pull request closed without merging**